Tests fail with Sphinx 7.1

```
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.11.4, pytest-7.4.0, pluggy-1.2.0 -- /home/dmitry/.virtualenvs/sphinx/bin/python3
cachedir: .pytest_cache
rootdir: /home/dmitry/upstream/sphinxcontrib-jquery
collected 5 items                                                                                                                                                                            

tests/test_jquery_installed.py::test_jquery_installed_sphinx_ge_60_use_sri FAILED                                                                                                      [ 20%]
tests/test_jquery_installed.py::test_jquery_installed_sphinx_ge_60 FAILED                                                                                                              [ 40%]
tests/test_jquery_installed.py::test_jquery_installed_sphinx_lt_60 SKIPPED (Requires Sphinx older than 6.0)                                                                            [ 60%]
tests/test_jquery_installed.py::test_integrity[jquery.js] PASSED                                                                                                                       [ 80%]
tests/test_jquery_installed.py::test_integrity[_sphinx_javascript_frameworks_compat.js] PASSED                                                                                         [100%]

========================================================================================== FAILURES ==========================================================================================
_________________________________________________________________________ test_jquery_installed_sphinx_ge_60_use_sri _________________________________________________________________________

blank_app = <function blank_app.<locals>.inner at 0x7f4c49146480>

    @pytest.mark.skipif(sphinx.version_info[:2] < (6, 0),
                        reason="Requires Sphinx 6.0 or greater")
    def test_jquery_installed_sphinx_ge_60_use_sri(blank_app):
        out_dir = blank_app(confoverrides={"extensions": ["sphinxcontrib.jquery"], "jquery_use_sri": True})
    
        text = out_dir.joinpath("index.html").read_text(encoding="utf-8")
>       assert ('<script '
                'integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" '
                'src="_static/jquery.js"></script>') in text
E       assert '<script integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" src="_static/jquery.js"></script>' in '<!DOCTYPE html>\n\n<html lang="en">\n  <head>\n    <meta charset="utf-8" />\n    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />\n\n    <title>&lt;no title&gt; &#8212; Python  documentation</title>\n    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=b3523f8e" />\n    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=039e1c02" />\n    <script integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" src="_static/jquery.js?v=5d32c60e"></script>\n    <script integrity="sha384-lSZeSIVKp9myfKbDQ3GkN/KHjUc+mzg17VKDN4Y2kUeBSJioB9QSM639vM9fuY//" src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>\n    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js?v=b3ba4146"></script>\n    <script src="_static/doctools.js?v=888ff710"></script>\n    <script src="_static/sphinx_highlight.js?v=4825356b"></script>\n    <link rel="index" title="Index" href="genindex.html" />\n    <link rel="search" title="Search" href="search.html" />\n   \n  <link rel="stylesheet" href="_static/cus... class="sphinxsidebarwrapper">\n<h1 class="logo"><a href="#">Python</a></h1>\n\n\n\n\n\n\n\n\n<h3>Navigation</h3>\n\n<div class="relations">\n<h3>Related Topics</h3>\n<ul>\n  <li><a href="#">Documentation overview</a><ul>\n  </ul></li>\n</ul>\n</div>\n<div id="searchbox" style="display: none" role="search">\n  <h3 id="searchlabel">Quick search</h3>\n    <div class="searchformwrapper">\n    <form class="search" action="search.html" method="get">\n      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>\n      <input type="submit" value="Go" />\n    </form>\n    </div>\n</div>\n<script>document.getElementById(\'searchbox\').style.display = "block"</script>\n\n\n\n\n\n\n\n\n        </div>\n      </div>\n      <div class="clearer"></div>\n    </div>\n    <div class="footer">\n      &copy;.\n      \n      |\n      Powered by <a href="http://sphinx-doc.org/">Sphinx 7.1.1</a>\n      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.13</a>\n      \n      |\n      <a href="_sources/index.rst.txt"\n          rel="nofollow">Page source</a>\n    </div>\n\n    \n\n    \n  </body>\n</html>'

tests/test_jquery_installed.py:41: AssertionError
_____________________________________________________________________________ test_jquery_installed_sphinx_ge_60 _____________________________________________________________________________

blank_app = <function blank_app.<locals>.inner at 0x7f4c48623ec0>

    @pytest.mark.skipif(sphinx.version_info[:2] < (6, 0),
                        reason="Requires Sphinx 6.0 or greater")
    def test_jquery_installed_sphinx_ge_60(blank_app):
        out_dir = blank_app(confoverrides={"extensions": ["sphinxcontrib.jquery"]})
    
        text = out_dir.joinpath("index.html").read_text(encoding="utf-8")
>       assert ('<script '
                'src="_static/jquery.js"></script>') in text
E       assert '<script src="_static/jquery.js"></script>' in '<!DOCTYPE html>\n\n<html lang="en">\n  <head>\n    <meta charset="utf-8" />\n    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />\n\n    <title>&lt;no title&gt; &#8212; Python  documentation</title>\n    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=b3523f8e" />\n    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=039e1c02" />\n    <script src="_static/jquery.js?v=5d32c60e"></script>\n    <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>\n    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js?v=b3ba4146"></script>\n    <script src="_static/doctools.js?v=888ff710"></script>\n    <script src="_static/sphinx_highlight.js?v=4825356b"></script>\n    <link rel="index" title="Index" href="genindex.html" />\n    <link rel="search" title="Search" href="search.html" />\n   \n  <link rel="stylesheet" href="_static/custom.css" type="text/css" />\n  \n  \n  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />\n\n  </head><body>\n  \n\n    <div cl... class="sphinxsidebarwrapper">\n<h1 class="logo"><a href="#">Python</a></h1>\n\n\n\n\n\n\n\n\n<h3>Navigation</h3>\n\n<div class="relations">\n<h3>Related Topics</h3>\n<ul>\n  <li><a href="#">Documentation overview</a><ul>\n  </ul></li>\n</ul>\n</div>\n<div id="searchbox" style="display: none" role="search">\n  <h3 id="searchlabel">Quick search</h3>\n    <div class="searchformwrapper">\n    <form class="search" action="search.html" method="get">\n      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>\n      <input type="submit" value="Go" />\n    </form>\n    </div>\n</div>\n<script>document.getElementById(\'searchbox\').style.display = "block"</script>\n\n\n\n\n\n\n\n\n        </div>\n      </div>\n      <div class="clearer"></div>\n    </div>\n    <div class="footer">\n      &copy;.\n      \n      |\n      Powered by <a href="http://sphinx-doc.org/">Sphinx 7.1.1</a>\n      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.13</a>\n      \n      |\n      <a href="_sources/index.rst.txt"\n          rel="nofollow">Page source</a>\n    </div>\n\n    \n\n    \n  </body>\n</html>'

tests/test_jquery_installed.py:59: AssertionError
================================================================================== short test summary info ===================================================================================
FAILED tests/test_jquery_installed.py::test_jquery_installed_sphinx_ge_60_use_sri - assert '<script integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" src="_static/jquery.js"></script>' in '<!DOCTYPE html>\n\n<html lang="en">\n  <head>\...
FAILED tests/test_jquery_installed.py::test_jquery_installed_sphinx_ge_60 - assert '<script src="_static/jquery.js"></script>' in '<!DOCTYPE html>\n\n<html lang="en">\n  <head>\n    <meta charset="utf-8" />\n    <meta name="viewport" content="width=device-width...
=========================================================================== 2 failed, 2 passed, 1 skipped in 1.33s ===========================================================================
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -30,10 +30,13 @@ jobs:
         - "4"  # jQuery included
         - "5"  # jQuery deprecated
         - "6"  # jQuery removed
+        - "7"
         exclude:
-        # Sphinx 6 does not support Python 3.7
+        # Sphinx >= 6 does not support Python 3.7
         - python: "3.7"
           sphinx: "6"
+        - python: "3.7"
+          sphinx: "7"
 
     steps:
     - uses: actions/checkout@v3

--- a/tests/test_jquery_installed.py
+++ b/tests/test_jquery_installed.py
@@ -38,12 +38,14 @@ def test_jquery_installed_sphinx_ge_60_use_sri(blank_app):
     out_dir = blank_app(confoverrides={"extensions": ["sphinxcontrib.jquery"], "jquery_use_sri": True})
 
     text = out_dir.joinpath("index.html").read_text(encoding="utf-8")
+    checksum = '?v=5d32c60e' if sphinx.version_info[:2] >= (7, 1) else ''
     assert ('<script '
             'integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" '
-            'src="_static/jquery.js"></script>') in text
+            f'src="_static/jquery.js{checksum}"></script>') in text
+    checksum = '?v=2cd50e6c' if sphinx.version_info[:2] >= (7, 1) else ''
     assert ('<script '
             'integrity="sha384-lSZeSIVKp9myfKbDQ3GkN/KHjUc+mzg17VKDN4Y2kUeBSJioB9QSM639vM9fuY//" '
-            'src="_static/_sphinx_javascript_frameworks_compat.js"></script>') in text
+            f'src="_static/_sphinx_javascript_frameworks_compat.js{checksum}"></script>') in text
 
     static_dir = out_dir / '_static'
     assert static_dir.joinpath('jquery.js').is_file()
@@ -56,10 +58,12 @@ def test_jquery_installed_sphinx_ge_60(blank_app):
     out_dir = blank_app(confoverrides={"extensions": ["sphinxcontrib.jquery"]})
 
     text = out_dir.joinpath("index.html").read_text(encoding="utf-8")
+    checksum = '?v=5d32c60e' if sphinx.version_info[:2] >= (7, 1) else ''
     assert ('<script '
-            'src="_static/jquery.js"></script>') in text
+            f'src="_static/jquery.js{checksum}"></script>') in text
+    checksum = '?v=2cd50e6c' if sphinx.version_info[:2] >= (7, 1) else ''
     assert ('<script '
-            'src="_static/_sphinx_javascript_frameworks_compat.js"></script>') in text
+            f'src="_static/_sphinx_javascript_frameworks_compat.js{checksum}"></script>') in text
 
     static_dir = out_dir / '_static'
     assert static_dir.joinpath('jquery.js').is_file()

